README.md

```diff
@@ -1,9 +1,9 @@
 # [macOS Setup Guide](https://sourabhbajaj.com/mac-setup)
 
-| branch | status |
+| workflows | status |
 | ------ | ------ |
-| `main` | [![main branch](https://github.com/sb2nov/mac-setup/workflows/Test/badge.svg)](https://github.com/sb2nov/mac-setup/actions) |
-| `health-check` | [![health-check branch](https://img.shields.io/travis/sb2nov/mac-setup/health-check.svg?label=links)](https://travis-ci.org/sb2nov/mac-setup) |
+| `Test` | [![Test](https://github.com/sb2nov/mac-setup/workflows/Test/badge.svg)](https://github.com/sb2nov/mac-setup/actions) |
+| `Links` | [![Links](https://github.com/sb2nov/mac-setup/workflows/Links/badge.svg)](https://github.com/sb2nov/mac-setup/actions) |
```

Comment:
The workflow name is now shown in the badge, could just show the badge and remove the table?


Could also link to the specific workflows rather than just `/actions`:

* https://github.com/sb2nov/mac-setup/actions?query=workflow%3ALinks
* https://github.com/sb2nov/mac-setup/actions?query=workflow%3ATest

Reply:
Updated it!